index.html

```diff
@@ -156,5 +156,18 @@ <h4 class="connect">LET'S CONNECT</h4>
             </div>
         </div>
     </section>
+    <div id="class_background">
+        <section class="contactMe" id="contact_back">
+            <h2>Contact me</h2>
+            <p class="contact_info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I'd love to help with it</p>
+            <form action="https://formspree.io/f/xqkoqyqb" method="post">
+                <input required type="text" name="user_name" maxlength="30" placeholder="Your name" class="form_info">
+                <input required type="mail" name="user_mail" placeholder="Your email" class="form_info">
+                <textarea required name="user_message" id="user_message" maxlength="500" placeholder="Write your message here..."></textarea>
+                <button class="resumeButton" type="button">Get in touch</button>
+            </form>
```

Comment:
- [ ] Nice work adding the required if. However, note that the type attribute is set to "button", which means that the button will not submit any form data. If you want the button to submit a form, you should set the type attribute to "submit".

- If you want the button to trigger an action on your website or application, you can use JavaScript to add an event listener to the button. but it's easier with the first option 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/220506510-6308eff3-b699-476b-aa76-2bfcd003553b.png)


- [ ] You did ade effort on the button, but it still doesn't follow the guideline. Check the figma guide presented below, with the state of your project. you'd agree with  me that the present submit color does not give a good user experience 

## Figma:
![image](https://user-images.githubusercontent.com/50990925/220507069-ecaf67f1-b982-453c-841f-717fdbc3212c.png)

## View:
![image](https://user-images.githubusercontent.com/50990925/220507112-39d15b63-10bf-47a3-9e18-84e856aafa46.png)